### PR TITLE
Add documentation for user status selector dialog

### LINKED ISSUE #3770: Add documentation for set user status
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

Add documentation for the new user status dialog

### PR DESCRIPTION
**Pull request by FlexW**
Fixes #3770

Signed-off-by: Felix Weilbach <felix.weilbach@nextcloud.com>

### CODE REVIEW

File: doc/navigating.rst [hunk 1]
```diff
@@ -157,6 +157,35 @@ icon.
 If a directory includes ignored files that are marked with warning icons
 that does not change the status of the parent directories.
 
+Set the user status
+-------------------
+
+If you have the user status app is installed on your Nextcloud server,
```
Comment (mgallien) [inline doc/navigating.rst:163]: you have an extra `is`
```
If you have the user status app installed on your Nextcloud server,
```

File: doc/navigating.rst [hunk 2]
```diff
@@ -157,6 +157,35 @@ icon.
...
+you can set your user status from the desktop client. To do so, open
+the main dialog. Then click on your avatar and then click on the tree
+dots. In the menu that now opened click on **Set status**.
```
Comment (mgallien) [inline doc/navigating.rst:166]: better
```
dots. In the menu that opens click on **Set status**.
```

File: doc/navigating.rst [hunk 3]
```diff
@@ -157,6 +157,35 @@ icon.
...
+
+.. figure:: images/user_status_selector_main_dialog.png
+   :alt: Open user status dialog from main dialog.
+
+In the dialog that will now open, you can set your online status if
```
Comment (mgallien) [inline doc/navigating.rst:171]: remove future usage
```
In the dialog that opens, you can set your online status if
```

File: doc/navigating.rst [hunk 4]
```diff
@@ -157,6 +157,35 @@ icon.
 If a directory includes ignored files that are marked with warning icons
 that does not change the status of the parent directories.
 
+Set the user status
+-------------------
+
+If you have the user status app installed on your Nextcloud server,
+you can set your user status from the desktop client. To do so, open
+the main dialog. Then click on your avatar and then click on the tree
```
Comment (mgallien) [inline doc/navigating.rst:165]: s/tree/three/

**Review by mgallien — CHANGES_REQUESTED** [review]
just a small last typo

File: doc/navigating.rst (hunk 4 above)
Comment (FlexW) [reply doc/navigating.rst:165]: thanks. done

Comment (mgallien) [reply doc/navigating.rst:165]: in the last commit it is still `tree`

Comment (FlexW) [reply doc/navigating.rst]: I forgot to push. Sorry.

**Review by mgallien — APPROVED** [review]

**Review by allexzander — APPROVED** [review]

**Pull request merged**